e2e/core/test.ts

```diff
@@ -33,3 +33,7 @@ export const test = base.extend<CustomTestFixtures, CustomWorkerFixtures>({
     { scope: 'test', auto: true },
   ],
 });
+
+test.afterEach(async ({ api, patient }) => {
+  if (patient.uuid) await deletePatient(api, patient.uuid);
+});
```

Comment:
I dug a bit deeper and saw that it's already handled when defining the fixture:
https://github.com/openmrs/openmrs-esm-patient-management/blob/75b586b623582c2c114baf9df733bf7760d79e47/e2e/core/test.ts#L28

That said, we should get rid of the try-catch there.

It should just be:
```
const patient = await generateRandomPatient(api);
await use(patient);
await deletePatient(api, patient.uuid);
```
No try catch, no if statements. Let the test fail.